src/main/java/edu/cornell/kfs/sys/CUKFSConstants.java

```diff
@@ -16,6 +16,8 @@
     public static final String RULE_CODE_CA = "CA";
     public static final String DELIMITER = ".";
     public static final String DATE_FORMAT_YYYY = "YYYY";
+    public static final String FILE_EXTENSION_DELIMITER = ".";
```

Comment:
I think the other DELIMITER constant was meant to be used in the context of property delimiters. If you think that the existing constant is also okay to use in the context of file delimiters, then I'll change this accordingly.